src/core/memory.cc

```diff
@@ -1641,6 +1641,13 @@ void *allocate_slowpath(size_t size) {
             return original_malloc_func(size);
         }
 
+        // original_malloc_func might be null for allocations before main
+        // in constructors before original_malloc_func ctor is called
+        // Note on #2137: Moved to here, because there is lots of code 
+        // that implicitly relies on the SIF below to have occurred, and thus
```

Comment:
What is SIF?

Reply:
guess it's static init fiasco .

Reply:
static init fiasco.

Reply:
probably, would be better to repeat "static init fiasco" here, before "SIF" becomes a well-known abbreviation, like, for instance, OOM.

Reply:
Sure.

Reply:
Arguably, even the pharse "static init fiasco" doesn't mean anything to the uninitiated.